ekf:
  output:
    topic_name: /estimate/ekf
  input:
    topic_name: /drone/out/ekf_input
  init_values:
    x0: [0, 0, 0, 0, 0, 0]
    P0: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
  dt: 0.02

measurements:
  dnn_cv_position:
    topic_name: /estimate/dnn_cv/position
    sigmas: [0.1, 0.1, 0.1] # std. dev. in x,y,z (m)
  drone_velocity:
    topic_name: /drone/out/telemetry
    sigmas: [0.1, 0.1, 0.1] # std. dev. in v_x, v_y, v_z (m/s)
  tcv_cv:
    topic_name: /estimate/drone_pose/tcv
    sigmas: [0.1, 0.1, 0.1, 0.1] # std. dev. in x,y,z,psi
